v63005/outref/gtm8941.txt

```diff
@@ -7,3 +7,13 @@
 %YDB-I-LOCKSPACEINFO, Region: DEFAULT: processes on queue: 0/160; LOCK slots in use: 0/120; name space not full
 %YDB-I-NOLOCKMATCH, No matching locks were found in DEFAULT
 %YDB-I-LOCKSPACEUSE, Estimated free lock space: 100% of 40 pages
+
+# Running the command $ydb_dist/lke show -nocrit
+%YDB-I-LOCKSPACEINFO, Region: DEFAULT: processes on queue: 0/160; LOCK slots in use: 0/120; name space not full
+%YDB-I-NOLOCKMATCH, No matching locks were found in DEFAULT
+%YDB-I-LOCKSPACEUSE, Estimated free lock space: 100% of 40 pages
+
+# Running the command $ydb_dist/lke show -nocritical
```

Comment:
Sorry. Forgot to mention. Can you test LKE CLEAR -CRITICAL and LKE CLEAR -NOCRITICAL as well.

Reply:
lke clear does not recognize crit

Reply:
We will fix the documentation then as it mentions LKE CLEAR supports CRIT/NOCRIT.